gsd-web/quasar.conf.js

```diff
@@ -9,6 +9,7 @@
 /* eslint-env node */
 const ESLintPlugin = require('eslint-webpack-plugin')
 const { configure } = require('quasar/wrappers');
+const buildSha = require('./src/boot/buildVersion');
```

Comment:
Similar to #51, we aren't injecting this into the boot sequence, so I suspect this file should live elsewhere.